docs/developer-guide/nodejs-api.md

```diff
@@ -16,10 +16,10 @@ While ESLint is designed to be run on the command line, it's possible to use ESL
     * [static version][eslint-version]
     * [static outputFixes()][eslint-outputFixes]
     * [static getErrorResults()][eslint-getErrorResults]
-    * [LintResult type](lintresult)
-    * [LintMessage type](lintmessage)
-    * [EditInfo type](editinfo)
-    * [Formatter type](formatter)
+    * [LintResult type](#-lintresult-type)
+    * [LintMessage type](#-lintmessage-type)
+    * [EditInfo type](#-editinfo-type)
+    * [Formatter type](#-formatter-type)
```

Comment:
Can we use reference links instead? There's already `[lintresult]: #-lintresult-type` at the bottom of this document, so this could be just:

```
[LintResult type][lintresult]
```

(the same for other three links)

Reply:
Actually, please disregard my previous comment. This fix works well, and that's most important at this moment since the links were broken.

We also need to fix some other links in the same document, and then we could align the link style consistently across the whole document.